aten/src/ATen/quantized/Quantizer.h

```diff
@@ -211,14 +213,14 @@ struct CAFFE2_API PerChannelAffineQuantizer : public AffineQuantizer {
     auto* other_per_channel_affine =
         static_cast<PerChannelAffineQuantizer*>(other.get());
     return scalar_type() == other_per_channel_affine->scalar_type() &&
-        scales() == other_per_channel_affine->scales() &&
-        zero_points() == other_per_channel_affine->zero_points() &&
+        scales().equal(other_per_channel_affine->scales()) &&
+        zero_points().equal(other_per_channel_affine->zero_points()) &&
         axis() == other_per_channel_affine->axis();
   }
 
  private:
-  const std::vector<double> scales_;
-  const std::vector<int64_t> zero_points_;
+  const Tensor scales_;
+  const Tensor zero_points_;
```

Comment:
you can remove `const` here as well. cc @ezyang  we don't have a way to enforce a Tensor to be const right?

Reply:
That's right. `const Tensor` here just means that you can't overwrite `scales_` variable which is mostly pointless.